lib/paramedic.js

```diff
@@ -271,9 +271,13 @@ ParamedicRunner.prototype.runLocalTests = function () {
     })
     .then(function(command) {
         self.setPermissions();
-        logger.normal('cordova-paramedic: running command ' + command);
 
-        return execPromise(command);
+        if (!self.config.runMainTests()) {
```

Comment:
This change would break local Android Appium run, I believe. See the comment on line 283.
We need to boot up an Android emulator for Appium to run tests on and skipping this command would skip starting the emulator.